base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -458,11 +467,29 @@ public Double getInstantFuelConsumption() {
         return SdlDataTypeConverter.objectToDouble(object);
     }
 
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.

```

Comment:
@kostyaBoss When using the RPC generator tool, on line 474 I got 7.1.0. I would replace 7.1 with 7.1.0.